doc/api/configuration.md

```diff
@@ -166,6 +206,28 @@ df = %sql SELECT * FROM languages
 type(df)
 ```
 
+
+## `dsn_filename`
+
+Default: `odbc.ini`
+
+Set the Path to the DSN file.
+
+```{code-cell} ipython3
+%config SqlMagic.dsn_filename
+```
+
+You can change it according to the
+path to your DSN file
+
+```{code-cell} ipython3
+%config SqlMagic.dsn_filename = "new_path_to_dsn.ini"
+%config SqlMagic.dsn_filename
```

Comment:
i think once we run `%config SqlMagic.dsn_filename = "new_path_to_dsn.ini"` it should display the new name automatically. We shouldn't have to run `%config SqlMagic.dsn_filename`. Please open an issue

Reply:
no config changes display the new value by default, this is expected